app/cdap/components/Snackbar/Components/Transition/index.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box, Typography } from '@material-ui/core';
+import CheckCircleOutlinedIcon from '@material-ui/icons/CheckCircleOutline';
+import CloseIcon from '@material-ui/icons/Close';
+import ErrorOutlineIcon from '@material-ui/icons/ErrorOutline';
+import { useStyles } from 'components/Snackbar/Components/Transition/styles';
+import { ITransitionProps } from 'components/Snackbar/Components/Transition/types';
+import T from 'i18n-react';
+import React from 'react';
+
+export default function({
+  handleClose,
+  isSuccess,
+  messageToDisplay,
+  actionType,
+}: ITransitionProps) {
+  const undoActivity = () => {
```

Comment:
can we rename to handleUndoOperaiont or handleUndoAction?

Reply:
Renamed to transitionAction

Reply:
Renamed to handleUndoOperation